cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CertUtils.java

```diff
@@ -165,9 +146,9 @@ public static CertAndKey keyStoreCertAndKey(Secret secret, String keyCertName) {
         return new CertAndKey(
                 decodeFromSecret(secret, Ca.SecretEntry.KEY.asKey(keyCertName)),
                 decodeFromSecret(secret, Ca.SecretEntry.CRT.asKey(keyCertName)),
-                new byte[]{},
-                decodeFromSecret(secret, Ca.SecretEntry.P12_KEYSTORE.asKey(keyCertName)),
-                decodeStringFromSecret(secret, Ca.SecretEntry.P12_KEYSTORE_PASSWORD.asKey(keyCertName))
+                null,
```

Comment:
Rather than passing null here, it looks like CertAndKey has a constructor that only takes 2 arguments.